tests/test_ecephys/test_tools_spikeinterface.py

```diff
@@ -35,24 +33,20 @@
 
 
 class TestAddElectricalSeriesWriting(unittest.TestCase):
-    @classmethod
-    def setUpClass(cls):
-        """Use common recording objects and values."""
-        cls.num_channels = 3
-        cls.sampling_frequency = 1.0
-        cls.durations = [3.0]  # 3 samples in the recorder
-        cls.test_recording_extractor = generate_recording(
-            sampling_frequency=cls.sampling_frequency, num_channels=cls.num_channels, durations=cls.durations
-        )
-
```

Comment:
I'm pretty sure we initialize `cls.test_recording_extractor` at the class level to avoid having to regenerate it again for every test, which should decrease the overall amount of time it takes to run this particular set of tests without altering their behavior (I don't think they modify the `test_recording_extractor` in place)